packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -2582,7 +2582,6 @@ Activate a WebAuthn Factor by verifying the attestation and client data.
 | attestation       | Base64-encoded attestation from the WebAuthn JavaScript call        | Body       | String   | TRUE     |
 | clientData        | Base64-encoded client data from the WebAuthn JavaScript call        | Body       | String   | TRUE     |
 | factorId          | `id` of the Factor returned from enrollment                         | URL        | String   | TRUE     |
-| stateToken        | [State token](#state-token) for the current transaction             | Body       | String   | TRUE     |
```

Comment:
This is inaccurate -- there is no state token used here, and it was mistakenly added originally.